### PR TITLE
Add Patient Information to Pregnancy edit form

### PR DESCRIPTION
**Pull request by charleshuang80**
-Added links to left hand side
-Added patient information

still need to:
-add special circumstances checkbox
-style/format form to use bootstrap grid columns better

### CODE REVIEW

File: app/models/pregnancy.rb [hunk 1]
```diff
@@ -28,6 +28,7 @@ class Pregnancy
 	field :age, type: Integer
 	field :city, type: String
 	field :state, type: String #ennumeration?
+	field :zip, type: String
```
Comment (colinxfleming) [inline app/models/pregnancy.rb]: +1 on zip being a string and not an int!

**Comment by colinxfleming** [comment]
Looks good on first pass! Will take an in-depth look tonight, but can tell you that any suggested changes will wind up being pretty marginal stuff.

**Comment by colinxfleming** [comment]
Thanks a ton for your hard work on this, as always!

File: app/views/pregnancies/edit.html.erb [hunk 2]
```diff
@@ -50,10 +63,48 @@
           </p>
         </div>
       </div>
+      <div class="patient-info col-md-8">
```
Comment (colinxfleming) [inline app/views/pregnancies/edit.html.erb]: Nice, these are going to be super easy to extract into partials.

**Comment by colinxfleming** [comment]
Looks like this takes care of the bulk of #93, #94, and #95, yeah? We'll still need to arrange, extract into partials, set up toggling, and make the update route work. (And tests!)

**Comment by colinxfleming** [comment]
I'm going to leave those three issues open until they're in their own partials, but this will make it pretty trivial.

Thanks a ton for tackling this @charleshuang80 !

**Comment by charleshuang80** [comment]
Yeah, I think that it has most of those partials tickets covered, but not all is completed yet. A few thoughts:
1. There are definitely still parts that are missing from each of those sections. (There are also sections missing according to the wireframes (many fewer fields so likely easier)
2. None of the fields have actually been hooked into the update method for the controller yet. I would be wary of breaking them up into the partials until that has been completed and we know it works. And then it would require some care to make sure the right locals are passed in but shouldn't be too difficult.
3. They are thus far one giant form. If we are breaking up the view into partials, we might want to think through if it's possible to break it into smaller forms.
4. Unless/until the partials will be used in other places, I'm not sure I see the value of breaking up the view/form. But that's just my opinion, it's likely you have a bigger picture view of how it'll be useful.

**Comment by colinxfleming** [comment]
Going down the line:

1. Copy that. I think you're right to flag this. My thinking is that we can get it pretty close but it will wind up being a matter for @lwaldsc and @NerdyGirl537 to figure out and get exactly right. 
2. I hear you on the update method not being hooked up yet, and on the merit of taking this slow. If worse comes to worse we can develop it as a monolith and then break into partials as an enhancement. But I'd like to wind up with partials for sure.  
3. I think you're right to flag that there are parts of this we can separate from general information (notes, for example, might be a great candidate for this). But let's see how it shakes out first and make a decision on that down the road. 
4. My reasoning for wanting to break them up is ease in finding what to edit and keeping filesize down -- there's going to be a lot going on in the views, and there's going to be enough tweaking and editing that we want to keep things as simple as possible. KISS is my guiding principle here, followed closely by single responsibility.

**Pull request merged**